.github/workflows/build_and_publish.yaml

```diff
@@ -8,8 +8,7 @@ jobs:
   build-docs:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2.3.3
+    - uses: actions/checkout@v2
```

Comment:
Have you tested this workflow in isolation to verify this change?
I'm pretty sure we experienced a few regressions on the versioning here ...

Reply:
I have not. How can I do that?

Reply:
you can hack the workflow in a branch to be triggered on `push` and verify that the output (e.g. the version) is correct and cosistent